Add Abbas Zaidi to Before I Die

AbbasZaidi11 opened the pull request:
I am adding Abbas Zaidi to Before I Die with images and text.

AbbasZaidi11 committed:
Add Abbas Zaidi to Before I Die

XanderRubio committed:
Update Contributors.json

Added a spaced into the word NewYork.

XanderRubio commented on src/Components/LocationMap/LocationMap.jsx:
Hello @AbbasZaidi11, thank you for your contribution. 

Could you please assist me by providing details of the changes or additions you made in the LocationMap.jsx file? As your initial pull request for adding "Before I Die" only involved four files, and this is an additional code related to enhancement, I would appreciate it if you could explain the additions you made for my understanding before I merge your request. 

Also, I want to compliment you on the great image and text you submitted as they show up great on the preview deployment. You will create the opportunity for yourself to travel to New York. Thank you!

AbbasZaidi11 replied:
Sorry, if there is any sort of confusion but I did not make any changes to the map component. Also thanks for letting me contribute.

XanderRubio replied:
Thank you for clarifying. When I reviewed your changes, I noticed some modifications related to the formatting of the file. I assume this occurred due to automatic formatting. The component works normally when viewing the preview deployment and I will double-check everything to ensure that everything is fine. Your contribution is fantastic, and I will proceed to merge it onto the site.

XanderRubio committed:
Update LocationMap.jsx

Adding back the LocationMap code that is on the main branch to avoid a future merge conflict.

XanderRubio commented:
@AbbasZaidi11 Your pull request has been successfully merged!!! Congratulations <img alt="Bottle with Popping Cork" width="25" height="25" src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Food/Bottle%20with%20Popping%20Cork.png"><img alt="Bottle with Popping Cork" width="25" height="25" src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Food/Bottle%20with%20Popping%20Cork.png"><img alt="Bottle with Popping Cork" width="25" height="25" src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Food/Bottle%20with%20Popping%20Cork.png"><img alt="Double Exclamation Mark" width="25" height="25" src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Symbols/Double%20Exclamation%20Mark.png"><img alt="Double Exclamation Mark" width="25" height="25" src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Symbols/Double%20Exclamation%20Mark.png"><img alt="Double Exclamation Mark" width="25" height="25" src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Symbols/Double%20Exclamation%20Mark.png">. Make it happen for venturing forth and visiting the cosmopolitan city of New York!

 If you haven't already, please go ahead and star <img alt="Glowing Star" width="25" height="25" src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Travel%20and%20places/Glowing%20Star.png"><img alt="Glowing Star" width="25" height="25" src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Travel%20and%20places/Glowing%20Star.png"><img alt="Glowing Star" width="25" height="25" src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Travel%20and%20places/Glowing%20Star.png">this repo and you are more than welcome to continually contribute to this project by assisting with our [open issues](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues), or adding your own new open issues, and additionally you can get an idea of ideas we have to continually grow this project further as technology and tech stack continually evolve by looking at the [ROADMAP](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/ROADMAP.md) for the project. Have a great day, and thank you for taking the time to contribute, Abbas! You can view your contribution now on the [live link](https://before-i-die-achievements.vercel.app/). Have a great rest of your day and I will be sending you an invite to be a member of the Before I Die Code now.
